src/spatial_ref/srs.rs

```diff
@@ -284,34 +284,25 @@ impl SpatialRef {
     }
 
     #[cfg(major_ge_3)]
-    pub fn name(&self) -> Result<String> {
+    pub fn name(&self) -> Option<String> {
         let c_ptr = unsafe { gdal_sys::OSRGetName(self.0) };
-        if c_ptr.is_null() {
-            return Err(_last_null_pointer_err("OSRGetName"));
-        }
-        Ok(_string(c_ptr))
+        _string(c_ptr)
     }
 
-    pub fn angular_units_name(&self) -> Result<String> {
+    pub fn angular_units_name(&self) -> Option<String> {
         let mut c_ptr = ptr::null_mut();
         unsafe { gdal_sys::OSRGetAngularUnits(self.0, &mut c_ptr) };
```

Comment:
Angular units are assumed to be static, so no need to free them?

Reply:
Owned by `OGRSpatialReference`.